manifests/k8s-spot-termination-handler/k8s-spot-termination-handler.yaml

```diff
@@ -66,7 +66,7 @@ spec:
       serviceAccountName: k8s-spot-termination-handler
       containers:
         - name: k8s-spot-termination-handler
-          image: kubeaws/kube-spot-termination-notice-handler:1.13.7-1
+          image: stafot/kube-spot-termination-notice-handler:1.21.0_628d3aef
```

Comment:
I remember , you mentioned that we eventually move it from stafot/* to  mm/ docker repo. right?